datasets/modeled/buildings/dataset.json5

```diff
@@ -0,0 +1,116 @@
+{
+  dataset_id: "decarb_2023_buildings",
+  dataset_type: "modeled",
+  sector_description: "",
+  description: "",
+  origin_creator: "Aven Satre Meloy",
+  origin_organization: "",
+  origin_contributors: [
+  ],
+  origin_project: "",
+  origin_date: "",
+  origin_version: "",
+  data_source: "",
+  source: "",
+  data_classification: "low", // TODO: confirm
+  tags: [
+  ],
+  trivial_dimensions: [
+    "scenario",
+    "subsector",
+    "weather_year",
+  ],
+  data_schema: {
+    table_format: {
+      format_type: "unpivoted",
+      value_column: "value",
+    },
+    data_schema_type: "one_table",
+  },
+
+  dimensions: [
+    {
+      type: "geography",
+      name: "Region",
+      display_name: "Region",
+      file: "dimensions/regions.csv",
+      module: "dsgrid.dimension.standard",
+      class: "GeographyDimensionBaseModel",
+      description: "",
+    },
+    {
+      type: "metric",
+      name: "Building End Uses",
+      display_name: "Building End Uses",
+      file: "dimensions/enduses.csv",
+      module: "dsgrid.dimension.standard",
+      class: "EnergyEndUse",
+      description: "",
+    },
+    {
+      type: "model_year",
+      name: "Model Year 2018",
+      display_name: "Model Year 2018",
+      file: "dimensions/model_years.csv",
+      module: "dsgrid.dimension.standard",
+      class: "ModelYear",
+      description: "Model year: 2018",
+    },
+    {
+      type: "scenario",
+      class: "Scenario",
+      name: "DECARB 2023 High-Resolution Demand Scenarios",
+      display_name: "High-Resolution Demand Scenarios",
+      description: "DECARB 2023 High-Resolution Demand Scenarios; Limited to a single, high decarbonization scenario",
+      file: "../../../project/dimensions/scenarios.csv",
+      module: "dsgrid.dimension.standard",
+    },
+    {
+      type: "sector",
+      name: "buildings-sectors",
```

Comment:
Should this just be "Buildings Sectors" (like the display_name)?

Reply:
A minor thing but I might suggest "Building Sectors".